src/sparseml/modifiers/obcq/utils/helpers.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 import logging
+from collections import defaultdict
```

Comment:
in general would it be possible to add docstrings/typing to this file? just so that it's easier to follow. not for this PR but would be nice to see

Reply:
Sure, will add it in a different PR, created a ticket to track request https://app.asana.com/0/1201735099598270/1205939915090873/f